src/styles/block-overrides.scss

```diff
@@ -24,6 +24,22 @@
 	}
 }
 
+.is-style-heading-callout {
+	position: relative;
+	margin-top: 3rem !important;
+	margin-bottom: 1.5rem !important;
```

Comment:
Are we getting rid of the !importants here or did you figure out why they were needed?